docs/internal_api/index.rst

```diff
@@ -5,9 +5,10 @@ Internal API
 ############
 
 This page shows already-implemented Uxarray internal API functions. You can also
-check the draft `UXarray API
-<https://github.com/UXARRAY/uxarray/blob/main/docs/user_api/uxarray_api.md>`_
-documentation to see the tentative whole API and let us know if you have any feedback!
+check the `UXarray Milestones <https://github.com/UXARRAY/uxarray/milestones>`_ and
+`UXarray RoadMap <https://github.com/orgs/UXARRAY/projects/2/views/17>`_ for a high
+level understanding of UXarray's future function development milestones and roadmap.
+Please let us know if you have any feedback!
```

Comment:
Could it be worth mentioning how they could provide feedback? Maybe a link to submitting an issue on our repo?

Reply:
Maybe not from under the API docs, but I think we should have such a guidance somewhere in our docs (I noticed there is no such information anywhere in the docs)